README.md

```diff
@@ -1014,8 +1014,6 @@ for outputs like
 
 1. The Logbook Servlet Filter interferes with downstream code using `getWriter` and/or `getParameter*()`. See [Servlet](#servlet) for more details.
 2. The Logbook Servlet Filter does **NOT** support `ERROR` dispatch. You're strongly encouraged to not use it to produce error responses.
-2. The Logbook HTTP Client integration is handling gzip-compressed response entities incorrectly if the interceptor runs before a decompressing interceptor. Since logging compressed contents is not

```

Comment:
Hi @kasmarian, what think about add logic for manage ir more intelligently? That is add a flag to enable this fuature, for example in DefaultLogbook all is unchanged, but if you don't care for performace in pos of easy log configuration, you can opt to take a new "GzipLogbook" that incorpore this behaviour.
The developers can choise which is better for yours proyects.

Reply:
I like the idea of the toggle. Probably a good place for it would be in `LogbookHttpResponseInterceptor` and `LogbookHttpAsyncResponseConsumer` constructors to be overloaded with this extra flag. They would pass it to `RemoteResponse` when creating it. And  `LogbookAutoConfiguration` would also pass the flag to the `LogbookHttpResponseInterceptor` while initializing it. 

The name of the property could be something like `logbook.httpclient.decompress-response` to make sure it's clear that it only affects the httpclient module. Please also don't forget to update the README with this config and `additional-spring-configuration-metadata.json` file, so that IDEs can suggest it.

Reply:
I commited a possible implementation of that you mentioned with the Readme modification.

Reply:
Thank you! The change looks good to me. Could you please revert the changes related to formatting? I think your IDE adjusted some spacing and removed spaces at the ends of some the files. I'll merge the change then.

Reply:
Of course.
It has been modified.